addons/l10n_account_withholding/wizards/account_payment_register_views.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="name">account.payment.register.form</field>
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="account.view_account_payment_register_form"/>
+        <field name="arch" type="xml">
+            <group name="group1" position="inside">
+                <field name="l10n_account_withholding_withhold_tax" invisible="not l10n_account_withholding_display_withholding"/>
+            </group>
+            <footer position="before">
+                <!-- Hide for argentina, it should be handled by their own module -->
+                <notebook name="payment_register_notebook" colspan="2" invisible="not l10n_account_withholding_withhold_tax or not can_edit_wizard or (can_group_payments and not group_payment) or country_code == 'AR'">
```

Comment:
It is a bit weird to do that.  Why not inherit the attributes in the AR module itself?  (in theory this module does not need to speak about AR)

Reply:
That would be counter productive imo, as the aim is to merge this, forward port, and later on make the ar module depend on this one while removing the duplicated logic in that one.

This module is aimed to be a base for all countries that need this feature (ar, ph, id, the at least) and then we can build on that in a localized module if needed to add country-specific stuff (withholding tax report/certificates/...)

This solution came from TSB, while it may seem weird to see AR mentioned here it's temporary and also it won't break anything.